<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>1030</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PLE scan</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::GroupedDragging</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Save"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>O&amp;ptions</string>
    </property>
    <addaction name="action_ple_settings"/>
    <addaction name="action_optimizer_settings"/>
    <addaction name="action_show_fit_settings"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionToggle_scan"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="action_optimize_position"/>
   <addaction name="actionTrackPLE"/>
   <addaction name="actionFull_range"/>
  </widget>
  <widget class="QDockWidget" name="scan_control_DockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>133</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>200</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Scan params</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="controlDockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="rangeLabel">
         <property name="text">
          <string>Scan range:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="ScienDSpinBox" name="startDoubleSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-99.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99999999999.990005493164063</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ScienDSpinBox" name="stopDoubleSpinBox">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-99.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000000000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="2" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="speedLabel">
           <property name="text">
            <string>Frequency:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="frequencyDoubleSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="value">
            <double>50.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="resLabel">
           <property name="text">
            <string>Resolution</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="resolutionDoubleSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>5000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="3" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="linesLabel">
           <property name="text">
            <string>Lines to scan:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="number_of_repeats_SpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>1000000000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="scannedLabel">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Scanned:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="elapsed_lines_DisplayWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="constLabel">
         <property name="text">
          <string>Current target:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ScienDSpinBox" name="constDoubleSpinBox">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100000000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Channel:</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="channel_comboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ple_data_dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>95</width>
     <height>248</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>PLE </string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QWidget" name="ple_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ple_matrix_dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>209</width>
     <height>311</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>PLE matrix</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="PlotWidget" name="voltscan_matrix_ViewWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="0">
            <widget class="QLabel" name="percentLabel1">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;% :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="percentLabel2">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;% :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QRadioButton" name="voltscan_cb_manual_RadioButton">
             <property name="text">
              <string>Man&amp;ual</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QRadioButton" name="voltscan_cb_centiles_RadioButton">
             <property name="text">
              <string>&amp;Centiles</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="ScienDSpinBox" name="voltscan_cb_max_InputWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ScienDSpinBox" name="voltscan_cb_high_centile_InputWidget"/>
           </item>
           <item row="3" column="1">
            <widget class="ScienDSpinBox" name="voltscan_cb_low_centile_InputWidget"/>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="ScienDSpinBox" name="voltscan_cb_min_InputWidget"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="PlotWidget" name="voltscan_cb_ViewWidget">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ple_averaged_dockWidget">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>PLE averaged</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QWidget" name="ple_averaged_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ple_retrace_data_dockWidget">
   <property name="windowTitle">
    <string>Retrace</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="ple_retrace_widget"/>
  </widget>
  <widget class="QDockWidget" name="retrace_matrix_dockWidget">
   <property name="windowTitle">
    <string>Retrace matrix</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <widget class="QDockWidget" name="avg_retrace_dockWidget">
   <property name="minimumSize">
    <size>
     <width>77</width>
     <height>38</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Retrace averaged</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
  <action name="action_ple_settings">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Users/yy3/artwork/icons/oxygen/22x22/configure.png</normaloff>../../../../../../Users/yy3/artwork/icons/oxygen/22x22/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Users/yy3/artwork/icons/oxygen/22x22/document-save.png</normaloff>../../../../../../Users/yy3/artwork/icons/oxygen/22x22/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save odmr data</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Users/yy3/artwork/icons/oxygen/22x22/application-exit.png</normaloff>../../../../../../Users/yy3/artwork/icons/oxygen/22x22/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_optimize_position">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Refocus</string>
   </property>
  </action>
  <action name="actionToggle_scan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle scan</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionFull_range">
   <property name="text">
    <string>Full range</string>
   </property>
  </action>
  <action name="action_optimizer_settings">
   <property name="text">
    <string>Optimizer</string>
   </property>
  </action>
  <action name="action_show_fit_settings">
   <property name="text">
    <string>Fit configuration</string>
   </property>
  </action>
  <action name="actionTrackPLE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Track PLE</string>
   </property>
  </action>
  <zorder>scan_control_DockWidget</zorder>
  <zorder>ple_data_dockWidget</zorder>
  <zorder>ple_matrix_dockWidget</zorder>
  <zorder>ple_averaged_dockWidget</zorder>
  <zorder>ple_retrace_data_dockWidget</zorder>
  <zorder>retrace_matrix_dockWidget</zorder>
  <zorder>avg_retrace_dockWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScienDSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qudi.util.widgets.scientific_spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>number_of_repeats_SpinBox</tabstop>
  <tabstop>resolutionDoubleSpinBox</tabstop>
  <tabstop>frequencyDoubleSpinBox</tabstop>
  <tabstop>constDoubleSpinBox</tabstop>
  <tabstop>voltscan_cb_max_InputWidget</tabstop>
  <tabstop>voltscan_cb_high_centile_InputWidget</tabstop>
  <tabstop>voltscan_cb_low_centile_InputWidget</tabstop>
  <tabstop>voltscan_cb_min_InputWidget</tabstop>
  <tabstop>voltscan_cb_centiles_RadioButton</tabstop>
  <tabstop>voltscan_cb_manual_RadioButton</tabstop>
  <tabstop>voltscan_cb_ViewWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
